test/membrane_hackney/sink_test.exs

```diff
@@ -2,14 +2,43 @@ defmodule Membrane.Element.Hackney.SinkTest do
   use ExUnit.Case, async: true
   use Mockery
 
+  import Membrane.Testing.Assertions
+
   alias Membrane.Buffer
+  alias Membrane.Element.CallbackContext, as: Ctx
+  alias Membrane.Testing.MockResourceGuard
 
   @module Membrane.Hackney.Sink
 
   @mock_url "http://some_url.com/upload"
   @mock_conn_ref :conn_ref
   @mock_payload "payload!"
 
+  defp get_contexts(_params) do
+    {:ok, resource_guard} = MockResourceGuard.start_link()
+
+    ctx_fields = [
+      playback: :playing,
+      pads: %{},
+      clock: nil,
+      parent_clock: nil,
+      resource_guard: resource_guard,
+      utility_supervisor: :mock_utility_supervisor,
+      name: :source
+    ]
+
+    [
+      ctx_write: Ctx.Write,
+      ctx_playing: Ctx.Playing,
+      ctx_init: Ctx.Init,
+      ctx_end_of_stream: Ctx.StreamManagement,
+      ctx_event: Ctx.Event
+    ]
+    |> Bunch.KVList.map_values(&struct(&1, ctx_fields))
```

Comment:
Do we need the pipe operator here?

Reply:
Do you mean to stop using `Bunch.KVList.map_values/2` or to write it without `|>`?

Reply:
I meant to write it without `|>`, but if you prefer that syntax, let's leave it that way